proto/incidents.proto

```diff
@@ -7,39 +7,38 @@
 
 syntax = "proto3";
 option optimize_for = LITE_RUNTIME;
-package valhalla.incidents;
+package valhalla;
 
 message IncidentsTile {
   // Sorted list of edge_ids describing what incidents are attached to an edge_id
   repeated Location locations = 1;
   // Look at `incident_locations` to find how to index this array
   repeated Metadata metadata = 2;
-}
```

Comment:
i removed the namespace (because the rest of the valhalla protos dont have sub namespaces) and i encapsulated the location and metadata defintions in the tile because location already exists elsewhere in the project and wuld be ambiguous. also there was inconsistent whitespace so i fixed that